guides/doc-Managing_Content/master.adoc

```diff
@@ -27,6 +27,8 @@ include::common/assembly_importing-content.adoc[leveloffset=+1]
 
 include::common/assembly_managing-application-life-cycles.adoc[leveloffset=+1]
 
+include::common/assembly_restricting-hosts-access-to-content.adoc[leveloffset=+1]
+
 include::common/assembly_managing-content-views.adoc[leveloffset=+1]
```

Comment:
This should go either before or after lifecycle environments and content views-- not in between.